prover/prover/src/prover.rs

```diff
@@ -148,7 +151,9 @@ impl JobReporter for ProverReporter {
                 self.rt_handle.block_on(async {
                     let result = self
                         .pool
-                        .access_storage().await.unwrap()
+                        .access_storage()
+                        .await
+                        .unwrap()
```

Comment:
`unwrap()` should ideally be used in places where the operation is infallible, e.g. `hex::decode("12ab")`. Otherwise, since we need to panic on error anyway, `expect("message")` should be used. For `Result`s resolved via `?`, context is added everywhere, as far as I can see.

Reply:
@popzxc We need your professional opinion. 

I was thinking we should only `?` on recoverable errors, `.expect()` unrecoverable errors, and `.unwrap()` infallible operations

Should we replace some `.unwrap`s with `.expect`s or context-ed `?` here?